helix-core/src/main/java/org/apache/helix/PropertyKey.java

```diff
@@ -214,6 +215,16 @@ public PropertyKey clusterConfig() {
           _clusterName, ConfigScopeProperty.CLUSTER.toString(), _clusterName);
     }
 
+
+    /**
+     * Get a property key associated with this Cloud configuration
+     * @return {@link PropertyKey}
+     */
+    public PropertyKey cloudConfig() {
+      return new PropertyKey(CONFIGS, ConfigScopeProperty.CLOUD, CloudConfig.class,
+          _clusterName, ConfigScopeProperty.CLOUD.toString(), _clusterName);
```

Comment:
Dont use toString(). Our convention will be use the enum's name() function. Also please change the previous one.

Reply:
Fixed.